### ISSUE TITLE
For Nigerian Wikipedia

### ISSUE THREAD

**Issue by tope** [edsu/wikichanges#15]
Could the Nigerian wikipedia for Nigerian parliament be added?

**Comment by tope** [comment]
Could the Nigerian wikipedia for Nigerian parliament be added?

**Comment by edsu** [comment]
Yes, in theory. Could you add a stanza to the JSON like this? 

https://github.com/edsu/wikichanges/blob/master/wikichanges.js#L859-879

**Comment by tope** [comment]
Can I edit the code and push back?

On 11 August 2015 at 21:00, Ed Summers <notifications@github.com> wrote:

> Yes, in theory. Could you add a stanza to the JSON like this?
>
> https://github.com/edsu/wikichanges/blob/master/wikichanges.js#L859-879
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edsu/wikichanges/issues/15#issuecomment-130046130>.
>



-- 
Tope

**Comment by edsu** [comment]
Yes, just fork it into your own space and send a pull request :-)
